packages/asset/contracts/OperatorFilter/interfaces/IOperatorFilterRegistry.sol

```diff
@@ -0,0 +1,105 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.18;
+
+interface IOperatorFilterRegistry {
```

Comment:
natspec

Reply:
why did we added it manually here then (and auditors asked us to add natspec btw)

Reply:
sol version maybe ?

Reply:
Yes we had Asset with solc compiler version 0.8.2 and the dependencies had a much higher solc compiler version;